invoice_fiscal_position_update/README.rst

```diff
@@ -0,0 +1,77 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============================
+invoice_fiscal_position_update
+==============================
+
+This module extends allow you to update invoice lines 
+and apply the fiscal position on lines that haven't been matched.'
+
```

Comment:
The 'description' in __openerp__.py was better than this one, so it may be better to copy-paste it here.